x-pack/legacy/plugins/apm/public/components/shared/SelectWithPlaceholder/index.tsx

```diff
@@ -31,11 +31,13 @@ export const SelectWithPlaceholder: typeof EuiSelect = props => {
       value={isEmpty(props.value) ? NO_SELECTION : props.value}
       onChange={e => {
         if (props.onChange) {
-          const customEvent = Object.assign(e, {
-            target: Object.assign(e.target, {
+          const customEvent = {
+            ...e,
+            target: {
+              ...e.target,
               value: e.target.value === NO_SELECTION ? '' : e.target.value
-            })
-          });
+            }
+          };
```

Comment:
@smith The problem of replacing it by spread operator is that it doesn't inherit the prototype properties, which means that e.preventDefault  is undefined. I was wondering to replace only the e.target.value.
const value = e.target.value === NO_SELECTION ? '' : e.target.value;
e.target.value = value;
WDYT?
<img width="1287" alt="Screenshot 2020-04-10 at 09 50 29" src="https://user-images.githubusercontent.com/55978943/78973707-bb486b00-7b10-11ea-9ceb-063cb2fdd191.png">